tests/tx/test_indexes.py

```diff
@@ -136,6 +136,35 @@ def test_genesis_not_in_mempool(self):
         for tx in self.genesis_txs:
             self.assertNotIn(tx, mempool_txs)
 
+    def test_deps_index(self):
+        from hathor.indexes.memory_deps_index import MemoryDepsIndex
+
+        add_new_blocks(self.manager, 5, advance_clock=15)
+        add_blocks_unlock_reward(self.manager)
+
+        # XXX: this test makes use of the internals of the memory deps-index implementation
+        deps_index: MemoryDepsIndex = self.manager.tx_storage.indexes.deps
```

Comment:
Shouldn't we test the RocksDB implementation as well?

Reply:
Done.

Reply:
Thanks!